stage1/init/init.go

```diff
@@ -110,8 +127,40 @@ func main() {
 	env = append(env, "LD_PRELOAD="+filepath.Join(path.Stage1RootfsPath(c.Root), "fakesdboot.so"))
 	env = append(env, "LD_LIBRARY_PATH="+filepath.Join(path.Stage1RootfsPath(c.Root), "usr/lib"))
 
-	if err := syscall.Exec(args[0], args, env); err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to execute nspawn: %v\n", err)
-		os.Exit(5)
+	if privNet {
+		n, err := networking.Setup(c.Manifest.UUID)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to setup network: %v\n", err)
+			return 6
```

Comment:
I think classes of codes is good if we can agree on classes. But that's a separate change.